### ISSUE TITLE
View Past Event Toggle

### ISSUE THREAD

**Issue by vungc** [BCStudentSoftwareDevTeam/celts#1270]
“View past event” toggle in the events list page untoggles when the term is changed. The toggle needs to be updated to stay toggled even when the term is changed. It also cannot be toggled at all for Fall 2020 and Spring 2021 (pictured below)

![image](https://github.com/BCStudentSoftwareDevTeam/celts/assets/112138502/cf5aa25f-4414-4ca9-ba52-381f746f5089)
![image](https://github.com/BCStudentSoftwareDevTeam/celts/assets/112138502/d8ed7cbf-9acb-4f63-98cd-f55219dfec61)

**Comment by Mbeweg** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [The “View past event” toggle](https://trello.com/c/VuoJ90qx/780-the-view-past-event-toggle)

**Comment by vungc** [comment]
So we saw this code on the event_list.html **var g_isPastTerm ={{"false" if selectedTerm.isFutureTerm or selectedTerm.isCurrentTerm else "true" }};** and figured out that the disabling of the toggle for the terms Fall 2020 and Spring 2021 were because they are past terms. We did not realize that the current term in the system was Summer 2021. We are leaving that alone since it is not an issue. We will now be focusing on making the toggle consistent throughout the current and future terms.
